README.md

```diff
@@ -11,3 +11,64 @@
 ```
 npm install -g yarn
 ```
+
+## Distributable Packages
+
+Following packages are distributed via [npm registry](https://www.npmjs.com/):
+
+| Package Name | Description |
+| ------------ | ----------- |
+| `@openshift/dynamic-plugin-sdk` | Allows loading, managing and interpreting plugins. |
+| `@openshift/dynamic-plugin-sdk-utils` | Provides Kubernetes and React utilities. |
+| `@openshift/dynamic-plugin-sdk-webpack` | Allows building plugin assets with webpack. |
+
+Each package is versioned and published independently from other packages.
+
+## Publishing Packages
```

Comment:
That's this ticket: https://issues.redhat.com/browse/HAC-768 -- I guess you can merge it with this stuff...

Reply:
Thanks! Updated PR description to mention this ticket.